src/core/mainloop/connection.c

```diff
@@ -1473,6 +1506,8 @@ connection_listener_new(const struct sockaddr *listensockaddr,
    */
   connection_check_oos(get_n_open_sockets(), 0);
 
+  log_notice(LD_NET, "Opened %s on %s",
+             conn_type_to_string(type), fmt_addrport(&addr, usePort));
```

Comment:
Should this maybe be at info log level instead?

Reply:
I think we have it at notice now, right?  We can leave it there if so, and move it to info later if we decide.

(If we don't have this at notice now, please ignore me)